one-ui/src/test/java/com/marklogic/hub/oneui/services/DataHubProjectUtilsTest.java

```diff
@@ -0,0 +1,63 @@
+package com.marklogic.hub.oneui.services;
+
+import com.marklogic.hub.curation.controllers.EnvironmentControllerTest;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class DataHubProjectUtilsTest {
+
+    @Test
+    public void testExtractZipFileWithoutArchiveFolder() throws Exception {
+        File file = new File(EnvironmentControllerTest.class.getClassLoader().getResource("dhfWithoutArchiveFolder.zip").getFile());
+        FileInputStream input = new FileInputStream(file);
+
+        Method method = DataHubProjectUtils.class.getDeclaredMethod("toByteArrayInputStream", InputStream.class);
```

Comment:
Just make these methods protected - that's common for unit-testing purposes.

Reply:
@rjrudin it is arguable though i do not feel it is good to do that. But it is harmless for me.

Reply:
Protected methods can still be changed between minor releases. You can also do package-level protection, which works as long as your test class is in the same package as your class.